peers/app_request_network.go

```diff
@@ -245,6 +254,25 @@ func (n *AppRequestNetwork) ConnectToCanonicalValidators(subnetID ids.ID) (*Conn
 	}, nil
 }
 
+func (n *AppRequestNetwork) Send(
+	msg message.OutboundMessage,
+	nodeIDs set.Set[ids.NodeID],
+	subnetID ids.ID,
+	allower subnets.Allower,
```

Comment:
do we need to set `allower` here? Or can we just set as the noop allower

Reply:
I'm open to either. In general I like to keep these wrapper interface methods as close as possible in the signature to the original but this is already broken here since we use the legacy VDR set instead of the new SendConfig. I'm fine with either:
:one: assuming our usage, since we only call send from here and do the conversion to SendConfig here and explicitly set the noopAllower here as well
:two: keep the interface of Send same as network.Send and do the conversion and allower setting in the application layer. 

I will defer on decision here to whatever the majority preference is